api/lib/domain/usecases/update-user-samlId.js

```diff
@@ -32,5 +34,10 @@ module.exports = async function updateUserSamlId({
     throw new UserAlreadyExistsWithAuthenticationMethodError();
   }
 
-  return userRepository.updateSamlId({ userId, samlId });
+  const authenticationMethod = new AuthenticationMethod({
```

Comment:
Vu en récap tech : Renommer le fichier en "add-gar-authentication-method-to-user"

Reply:
Done